docs/schemas/config-v1.schema.json

```diff
@@ -979,7 +979,7 @@
                             "properties": {
                                 "enabled": {
                                     "title": "/log/demux/<name>/enabled",
-                                    "description": "Indicates whether this demuxer will be used at the demuxing stage",
+                                    "description": "Indicates whether this demuxer will be used at the demuxing stage (defaults to 'true')",
```

Comment:
Note that this file is actually generated from code in `lnav_config.cc`.  I've pushed a change that reflects what you've done here.